src/applications/proxy-rewrite/partials/crisis-line-modal.js

```diff
@@ -0,0 +1,101 @@
+export default `
```

Comment:
Plain HTML version of the Veterans Crisis Line modal (used for desktop and mobile)